### PR TITLE
ticdc: add documentation on integration with Confluent

### PR DESCRIPTION
**Pull request by CharLotteiu**
### What is changed, added or deleted? (Required)

 - Add how to create an Avro changefeed.
 - Add a quick start guide for integrating with Confluent Platform.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/4102
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: ticdc/integrate-confluent-using-ticdc.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+[Confluent Platform](https://docs.confluent.io/current/platform.html) 是一个以 Kafka 为核心的流数据处理平台。它可以借助官方或第三方的 sink connector 将数据源连接到关系型或非关系型数据库中。
+
+你可以使用遵循 Avro 协议的 TiCDC 组件来集成 TiDB 和 Confluent Platform。TiCDC 可以将数据变化以 Confluent Platform 能识别的格式流式传输到 Kafka。下文详细介绍了集成的操作步骤。
+
+## 环境准备
+
+> **注意：**
+>
+> 在本教程中，[JDBC sink connector](https://docs.confluent.io/current/connect/kafka-connect-jdbc/sink-connector/index.html#load-the-jdbc-sink-connector) 用于将 TiDB 的数据复制到下游的关系型数据库中。为了简化操作，此处以 SQLite 为例。
```
Comment (TomShawn) [inline ticdc/integrate-confluent-using-ticdc.md:22]:
```suggestion
> 本教程使用 [JDBC sink connector](https://docs.confluent.io/current/connect/kafka-connect-jdbc/sink-connector/index.html#load-the-jdbc-sink-connector) 将 TiDB 的数据同步到下游的关系型数据库中。为了简化操作，此处以 SQLite 为例。
```

File: ticdc/integrate-confluent-using-ticdc.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
++ 确保 Zookeeper、Kafka 和 Schema Registry 已正确安装。推荐参照 [Confluent Platform 快速入门指南](https://docs.confluent.io/current/quickstart/ce-quickstart.html#ce-quickstart)部署本地测试环境。
+
++ 确保通过以下命令安装 JDBC sink connector。返回结果中需包含 `jdbc-sink`。
```
Comment (TomShawn) [inline ticdc/integrate-confluent-using-ticdc.md:26]:
```suggestion
+ 确保通过以下命令安装 JDBC sink connector。返回结果中预期包含 `jdbc-sink`。
```

File: ticdc/integrate-confluent-using-ticdc.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    curl -X POST -H "Content-Type: application/json" -d jdbc-sink-connector.json http://127.0.0.1:8083/connectors
+    ```
+
+3. 通过以下任一方式部署 TiCDC。如果已经部署了 TiCDC，可以跳过这一步。
+
+    - [Deploy and install TiCDC using TiUP](/ticdc/manage-ticdc.md#使用-tiup-部署安装-ticdc)
+    - [Use Binary](/ticdc/manage-ticdc.md#在原有-tidb-集群上使用-binary-部署-ticdc-组件)
```
Comment (TomShawn) [inline ticdc/integrate-confluent-using-ticdc.md:67]:
```suggestion
    - [使用 TiUP 部署安装 TiCDC](/ticdc/manage-ticdc.md#使用-tiup-部署安装-ticdc)
    - [在原有 TiDB 集群上使用 binary 部署 TiCDC 组件](/ticdc/manage-ticdc.md#在原有-tidb-集群上使用-binary-部署-ticdc-组件)
```

File: ticdc/integrate-confluent-using-ticdc.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+    在继续接下来的操作之前，请先确保 TiDB 和 TiCDC 集群可以正常运行。
+
+4. 运行下面的 `cdc cli` 命令，新建一个同步任务 `changefeed` ：
```
Comment (TomShawn) [inline ticdc/integrate-confluent-using-ticdc.md:71]:
```suggestion
4. 运行下面的 `cdc cli` 命令，新建一个同步任务 `changefeed`：
```

File: ticdc/integrate-confluent-using-ticdc.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+    ```shell
+    ./cdc cli changefeed create --pd="http://127.0.0.1:2379" --sink-uri="kafka://127.0.0.1:9092/testdb_test?protocol=avro" --opts "registry=http://127.0.0.1:8081"
+    ```
+
+    > **注意：**
+    >
+    > 请确保 PD、Kafka 和 Schema Registry 在各自的默认端口上运行。
+
+## 数据复制测试
```
Comment (TomShawn) [inline ticdc/integrate-confluent-using-ticdc.md:83]:
```suggestion
## 数据同步测试
```

File: ticdc/integrate-confluent-using-ticdc.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+
+TiDB 与 Confluent Platform 成功集成后，你可以按照以下步骤来测试数据复制功能。
```
Comment (TomShawn) [inline ticdc/integrate-confluent-using-ticdc.md:85]:
```suggestion
TiDB 与 Confluent Platform 成功集成后，你可以按照以下步骤来测试数据同步功能。
```

File: ticdc/integrate-confluent-using-ticdc.md [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+
+1. 在 TiDB 集群中新建 `testdb` 数据库：
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    CREATE DATABASE IF NOT EXISTS testdb;
+    ```
+
+    Create the `test` table in `testdb`:
```
Comment (TomShawn) [inline ticdc/integrate-confluent-using-ticdc.md:95]:
```suggestion
    在 `testdb` 数据库中创建 `test` 数据表：
```

File: ticdc/integrate-confluent-using-ticdc.md [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+        id INT PRIMARY KEY,
+        v TEXT
+    );
+    ```
+
+    > **注意：**
+    >
+    > 如果你需要修改数据库名或数据表名，要相应地修改 `jdbc-sink-connector.json` 文件中 `topics` 的参数值。
+
+2. 向 TiDB 数据库中插入数据：
```
Comment (TomShawn) [inline ticdc/integrate-confluent-using-ticdc.md:111]:
```suggestion
2. 向 `test` 数据表中插入数据：
```

File: ticdc/integrate-confluent-using-ticdc.md [hunk 9]
```diff
@@ -0,0 +1,129 @@
...
+    {{< copyable "sql" >}}
+
+    ```sql
+    INSERT INTO test (id, v) values (1, 'a');
+    INSERT INTO test (id, v) values (2, 'b');
+    INSERT INTO test (id, v) values (3, 'c');
+    INSERT INTO test (id, v) values (4, 'd');
+    ```
+
+3. 等待数据被复制到下游数据库中，并检查下游的数据：
```
Comment (TomShawn) [inline ticdc/integrate-confluent-using-ticdc.md:122]:
```suggestion
3. 等待数据被同步到下游数据库中，并检查下游的数据：
```

File: ticdc/manage-ticdc.md [hunk 10]
```diff
@@ -199,6 +199,25 @@ URI 中可配置的的参数如下：
...
+配置样例如下所示：
+
+{{< copyable "shell-regular" >}}
+
+```shell
+--sink-uri="kafka://127.0.0.1:9092/cdc-test?kafka-version=2.4.0&protocol=avro&partition-num=6&max-message-bytes=67108864&replication-factor=1"
+--opts registry="http://127.0.0.1:8081"
+```
+
+如果使用 Confluent 提供的 [data connectors](https://docs.confluent.io/current/connect/managing/connectors.html) 来向关系型或非关系型数据库传输数据，需要选择 `avro` 协议，并在 `opts` 中提供 [Confluent Schema Registry](https://www.confluent.io/product/confluent-platform/data-compatibility/) 的 URL。请注意，`avro` 协议和集成 Confluent 目前均为**实验特性**。
```
Comment (TomShawn) [inline ticdc/manage-ticdc.md:217]:
```suggestion
如要使用 Confluent 提供的 [data connectors](https://docs.confluent.io/current/connect/managing/connectors.html) 向关系型或非关系型数据库传输数据，应当选择 `avro` 协议，并在 `opts` 中提供 [Confluent Schema Registry](https://www.confluent.io/product/confluent-platform/data-compatibility/) 的 URL。请注意，`avro` 协议和 Confluent 集成目前均为**实验特性**。
```

File: ticdc/integrate-confluent-using-ticdc.md [hunk 11]
```diff
@@ -0,0 +1,129 @@
...
+    USE testdb;
+    CREATE TABLE test (
+        id INT PRIMARY KEY,
+        v TEXT
+    );
+    ```
+
+    > **注意：**
+    >
+    > 如果你需要修改数据库名或数据表名，要相应地修改 `jdbc-sink-connector.json` 文件中 `topics` 的参数值。
```
Comment (TomShawn) [inline ticdc/integrate-confluent-using-ticdc.md:109]:
```suggestion
    > 如果需要修改数据库名或数据表名，要相应地修改 `jdbc-sink-connector.json` 文件中 `topics` 的参数值。
```

File: ticdc/integrate-confluent-using-ticdc.md [hunk 12]
```diff
@@ -0,0 +1,129 @@
...
+
+本文档介绍如何使用 [TiCDC](/ticdc/ticdc-overview.md) 将 TiDB 集成到 Confluent Platform。
+
+> **警告：**
+>
+> 当前该功能为实验特性，请勿在生产环境中使用。
+
+[Confluent Platform](https://docs.confluent.io/current/platform.html) 是一个以 Kafka 为核心的流数据处理平台，可以借助官方或第三方的 sink connector 将数据源连接到关系型或非关系型数据库。
+
+你可以使用 TiCDC 组件和 Avro 协议来集成 TiDB 和 Confluent Platform。TiCDC 可以将数据更改以 Confluent Platform 能识别的格式流式传输到 Kafka。下文详细介绍了集成的操作步骤。
```
Comment (TomShawn) [inline ticdc/integrate-confluent-using-ticdc.md:16]:
```suggestion
你可以使用 TiCDC 组件和 Avro 协议来集成 TiDB 和 Confluent Platform。TiCDC 能将数据更改以 Confluent Platform 能识别的格式流式传输到 Kafka。下文详细介绍了集成的操作步骤。
```

**Comment by TomShawn** [comment]
LGTM
@liuzix PTAL

File: ticdc/integrate-confluent-using-ticdc.md [hunk 13]
```diff
@@ -0,0 +1,129 @@
...
+        "connection.url": "sqlite:test.db",
+        "connection.ds.pool.size": 5,
+        "table.name.format": "test",
+        "auto.create": true,
+        "auto.evolve": true
+      }
+    }
+    ```
+
+2. 运行下方命令新建一个 JDBC sink connector 实例（假设 Kafka 监听的 IP 地址是 `127.0.0.1:8083`）：
```
Comment (liuzix) [inline ticdc/integrate-confluent-using-ticdc.md:56]:
```suggestion
2. 运行下方命令新建一个 JDBC sink connector 实例（假设 Kafka 监听的 IP 地址与端口是 `127.0.0.1:8083`）：
```

File: ticdc/integrate-confluent-using-ticdc.md [hunk 14]
```diff
@@ -0,0 +1,129 @@
...
+    ```shell
+    curl -X POST -H "Content-Type: application/json" -d jdbc-sink-connector.json http://127.0.0.1:8083/connectors
+    ```
+
+3. 通过以下任一方式部署 TiCDC。如果已经部署了 TiCDC，可以跳过这一步。
+
+    - [使用 TiUP 部署安装 TiCDC](/ticdc/manage-ticdc.md#使用-tiup-部署安装-ticdc)
+    - [在原有 TiDB 集群上使用 binary 部署 TiCDC 组件](/ticdc/manage-ticdc.md#在原有-tidb-集群上使用-binary-部署-ticdc-组件)
+
+    在继续接下来的操作之前，请先确保 TiDB 和 TiCDC 集群可以正常运行。
```
Comment (liuzix) [inline ticdc/integrate-confluent-using-ticdc.md:69]:
```suggestion
    在继续接下来的操作之前，请先确保 TiDB 和 TiCDC 集群处于健康状态。
```

**Review by TomShawn — APPROVED** [review]

**Comment by liuzix** [comment]
LGTM

**Pull request merged**